test-configuration/test_config.py

```diff
@@ -3,4 +3,9 @@
     'disable_existing_loggers': True
 }
 
+PLUGINS = [
+    'netbox.tests.dummy_plugin',
```

Comment:
Is this a NetBox-provided test plugin?

Reply:
Yes, can be found here: https://github.com/netbox-community/netbox/tree/main/netbox/netbox/tests/dummy_plugin